lib/src/rules/avoid_returning_null_for_void.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r"Don't return null for void.";
+
+const _details = r'''
+
+**Don't** return null for void
```

Comment:
maybe replace w/ 

    **AVOID** returning null for void.

(I think that's more consistent on a quick scan.)

Reply:
Also, it would be great to have a sentence or two of rationale...